src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,80 @@
+@function week-width($day-width, $gap-between-days) {
+  @return $day-width * 7 + $gap-between-days * 6 + 20px;
+}
+
+$week-days:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1px;
+  width: week-width($day-width, $gap-between-days);
+  padding: 110px 10px 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-width;
+    height: $day-height;
```

Comment:
We need to add one more property, so that when the month starts on some other day than Monday we don't get that slight misalignment of the first week:
```
    box-sizing: border-box;
```